views/modify_schematisation_dialogs.py

```diff
@@ -134,17 +120,10 @@ def setupUi(self):
         self.buttonBox.setObjectName("buttonBox")
         self.verticalLayout.addWidget(self.buttonBox)
 
-        self.retranslateUi()
```

Comment:
Het lijkt erop dat `retranslateUi()` niet alleen verantwoordelijk is voor het vertalen van de UI, maar ook voor het zetten van (initiële) titels en dialogs...
Bij een kleine test waarbij ik deze heb uitgecommentariseerd verdwijnen allerlei titels/textboxen
![image](https://user-images.githubusercontent.com/11858775/58407594-70467600-806c-11e9-8606-f44e6541782b.png)

Reply:
Dit `retranslateUi` gebeurt op meerdere plekken waarbij het probleem waarschijnlijk ook is.

Reply:
Ik heb de setText/setWindowTitle zaken van dit PR teruggezet (zonder _translate). *Zou* weer moeten werken.